src/intersection/anthony-intersection-solution/intersection.js

```diff
@@ -0,0 +1,10 @@
+// Function to remove duplicates on the array
+const removeDuplicate = (arr) => arr.filter((e, i) => arr.indexOf(e) === i);
+// Function to return an array of values shared between two arrays
+export const intersection = (arr1 = [2, 1, 2], arr2 = [2, 3]) => {
```

Comment:
update you varibles their are not clear also the param should be empty array's to allow inputes